packages/charting/src/components/VerticalBarChart/__snapshots__/VerticalBarChart.test.tsx.snap

```diff
@@ -43,13 +43,29 @@ exports[`VerticalBarChart renders VerticalBarChart correctly 1`] = `
             {
               transform: translate(30px, 350px);
             }
+            & text {
+              -moz-osx-font-smoothing: grayscale;
+              -webkit-font-smoothing: antialiased;
+              fill: #605e5c;
+              font-family: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
+              font-size: 14px;
+              font-weight: 400;
+            }
       />
       <g
         className=
 
             {
               transform: translate(23px, 0px);
             }
+            & text {
+              -moz-osx-font-smoothing: grayscale;
+              -webkit-font-smoothing: antialiased;
+              fill: #605e5c;
+              font-family: 'Segoe UI', 'Segoe UI Web (West European)', 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif;
```

Comment:
use this from fabric, here and other places. also the fill color get that from themes

Reply:
HI @Raghurk ,

This snap file auto generated after snapshot test in vertical stacked bar chart. Because of that reason, styles load like above.